packages/react-scripts/template/README.md

```diff
@@ -1452,6 +1452,7 @@ If this doesn’t happen, try one of the following workarounds:
 * Some editors like Vim and IntelliJ have a “safe write” feature that currently breaks the watcher. You will need to disable it. Follow the instructions in [“Working with editors supporting safe write”](https://webpack.github.io/docs/webpack-dev-server.html#working-with-editors-ides-supporting-safe-write).
 * If your project path contains parentheses, try moving the project to a path without them. This is caused by a [Webpack watcher bug](https://github.com/webpack/watchpack/issues/42).
 * On Linux and macOS, you might need to [tweak system settings](https://webpack.github.io/docs/troubleshooting.html#not-enough-watchers) to allow more watchers.
+* Inside virtual machines such as (a Vagrant provisioned) VirtualBox, use `npm i --save-dev cross-env` and then [`cross-env CHOKIDAR_USEPOLLING=true npm start`](https://github.com/facebookincubator/create-react-app/issues/1049#issuecomment-261731734).  On Linux / macOS you can just use `CHOKIDAR_USEPOLLING=true npm start`.
```

Comment:
> I wouldn't mention Linux / macOS here at all.  Just because the person doing this uses macOS, doesn't mean a new team member won't need this to work on Windows at some point.

Ok.  Removed.  :)

> Let's rephrase as "run npm install [...] and then replace react-scripts start in package.json with cross-env [...].

Will try to re-parse your request tomorrow when brain's refreshed and come back with questions if I'm still stuck and finding it ambiguous.  At the moment I think you're ask me to update other parts of this README and [this line in package.json](https://github.com/facebookincubator/create-react-app/blob/master/package.json#L10) to be cross platform?  If so I'd prefer to open a separate issue and work on it there unless I'm misconstrued your request.